catalog-rest-service/src/main/resources/ui/src/components/Explore/Explore.component.tsx

```diff
@@ -414,6 +414,17 @@ const Explore: React.FC<ExploreProps> = ({
     setCurrentPage(1);
   }, [tab]);
 
+  useEffect(() => {
+    if (!isMounting.current) {
+      if (location.pathname === '/explore') {
```

Comment:
Why do we need this condition, and if we do let's use Route constants instead of hard-coded strings

Reply:
we need it because if I am on explore page and go to the dashboard or some other tab except a table, and I change filter let say I check on Tier 1, and I click again to explore page, its request API with filters for initial data.

Reply:
I will add constant for it